src/main.cpp

```diff
@@ -4,5 +4,6 @@ int main(int argc, char **argv)
 {
     auto app = Gtk::Application::create("org.gtk.wayfire-config-manager");
     std::unique_ptr<WCM> wcm = std::make_unique<WCM>(app);
+    app->signal_activate().connect([&] { wcm->window->present(); });
```

Comment:
I think it's better to put this into WCM constructor because `app->signal_command_line` and `app->signal_startup` actions are also there
And also it allows to keep `window` private

Reply:
Thanks.